concepts/05 Widgets/DataGrid/20 Editing/40 Customize Editors.md

```diff
@@ -62,6 +62,93 @@ The columns's [dataType](/api-reference/_hidden/GridBaseColumn/dataType.md '/Doc
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ...
+            @editor-preparing="onEditorPreparing">
+            <DxColumn
+                data-field="Note"
+                :editor-options="textAreaOptions"
+            />
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid, {
+        DxColumn
+    } from 'devextreme-vue/data-grid';
+    import 'devextreme-vue/text-area';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxColumn
+        },
+        data() {
+            return {
+                textAreaOptions: { height: 200 }
+            }
+        },
+        methods: {
+            onEditorPreparing(e) {
+                if(e.dataField == "Note") {
```

Comment:
Add the `&& e.parentType === "dataRow"` condition to avoid applying this logic to filter row if customer uses it. It will be very strange see dxTextEditor in filter row. Also the call e.setValue on onValueChanged callback will break debounce logic for filter row editor.